client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -107,13 +107,19 @@ module.exports = React.createClass( {
 	},
 
 	getCheckoutCompleteRedirectPath: function() {
-		var renewalItem;
+		var renewalItem, path;
 		if ( cartItems.hasRenewalItem( this.props.cart ) ) {
 			renewalItem = cartItems.getRenewalItems( this.props.cart )[ 0 ];
-			return purchasePaths.managePurchaseDestination( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId, 'thank-you' );
+			path = purchasePaths.managePurchaseDestination( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId, 'thank-you' );
```

Comment:
Our linter is set to discourage elses after returns. Putting the return back gives:
`114:10  warning  Unexpected 'else' after 'return'  no-else-return`